tmt/cli.py

```diff
@@ -1351,7 +1351,7 @@ def stories_id(
 @click.pass_context
 @click.argument('path', default='.')
 @option(
-    '-t', '--template', default='empty', metavar='TEMPLATE',
+    '-t', '--template', default='empty',
     type=click.Choice(['empty', *tmt.templates.INIT_TEMPLATES]),
     help=f"Template ({listed(tmt.templates.INIT_TEMPLATES, join='or')}).")
```

Comment:
I guess we should also adjust the `help` string not to duplicate the list:

```
Options:
  -t, --template empty|mini|base|full
                                  Template (mini, base or full).
```

Reply:
Agreed, fixed in a060c5ee

Reply:
Thanks!